frontend/src/pages/login/login.loader.ts

```diff
@@ -7,7 +7,7 @@ export type LoginLoaderData = object;
  * Loader data can be obtained using `useLoaderData()` in LoginPage.
  */
 export async function loginLoader(): Promise<LoginLoaderData> {
-  await ensureCSRF();
+  await ensureCSRFToken();
   const oidcInfo = await getOIDCInfo();
 
   return { oidcInfo: oidcInfo };
```

Comment:
This one is left because in this way when we land on the application the csrf token is retrieved already for future requests?

Reply:
That one is left because I forgot to remove it